tools/pd-analysis/transfer_region_counter.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pdAnalysis
```

Comment:
maybe just `analysis` according to https://blog.golang.org/package-names.  cc @rleungx

Reply:
thanks, done. @nolouch  @rleungx

Reply:
BTW, personally, I think this can be a binary so that we can use it to analysis PD's log.

Reply:
Do you mean that add parser for log and to run directly from cmd? If so, I will try to do it by new pr @rleungx

Reply:
Yes